### DISCUSSION TITLE
Fdroid link is down

### DISCUSSION THREAD

**Discussion by ippocratis** [Radiokot/photoprism-android-client#24] [General]
I don't use the fdroid builds
I prefer the GitHub ones
But I noticed that even though the server builds comple successfully,  the link leads to a "page not found" page

https://monitor.f-droid.org/builds/log/ua.com.radiokot.photoprism/6#site-footer

https://f-droid.org/packages/ua.com.radiokot.photoprism/

Opened an issue there

https://gitlab.com/fdroid/fdroid-website/-/issues/726

**Comment by Radiokot** [comment]
Thanks for paying attention. The app was added to the F-Droid repo only yesterday, it takes time for their website to update. One the app is available, I'll remove the ⌛️ prefix.
